index.html

```diff
@@ -56,7 +56,9 @@ <h2 class="section-header">Us!</h2>
                     -->
                     
                     <!-- Person #1 write your code between here-->
-                    
+                    <img src="https://cdn2.iconfinder.com/data/icons/hands-gestures/512/Gesture_Hand_Palm_Wave_Waving_Bye_Wavinghand1-512.png">
+                    <p>Hi my name is Yves, and check out my coolio portfolio</p>
```

Comment:
wrap everything but the `<img>` tag in a `<div>` with class `person-content` please!